apps/evisa/index.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+/* eslint-disable comma-dangle */
+
+const config = require('../../config');
+const agentEmail = require('./behaviours/agent-email')(config.email);
+const limitDocument = require('./behaviours/limit-documents');
+const removeImage = require('./behaviours/remove-image');
+const saveImage = require('./behaviours/save-image');
+
+module.exports = {
+  name: 'evisa',
+  baseUrl: '/',
+  steps: {
+    '/start': {
+      next: '/biometric-residence-permit-number'
+    },
+    '/biometric-residence-permit-number': {
+      fields: [
+        'biometric-residence-permit-number-options',
+        'biometric-residence-permit-number-number'
+      ],
+      next: '/start1'
+    },
+
+    '/start1': {
+      next: '/start2',
```

Comment:
these step slugs are made up from when I wanted to code the project, as BA requirement gathering was not finished yet, these should be modified to fit with what's on the design, but it is okay as the start up repo of the project